content/documents/implementing-single-sign-on-with-google-idp.mdx

```diff
@@ -0,0 +1,413 @@
+    | `ConnectClientSecret`   | This is the clientSecret of the identity provider (in this case Google)                                                                                                                                                    |
+    | `AppStartUrl`           | This is where the user will be redirected to after successful authentication via OpenID Connect. The parameter `{0}` represents the token. The parameter `{1} represents the IDP token, and the parameter {2} which is not used here represents the app start path used for deep linking`                                                                                  |
+    | `AuthorizationEndpoint` | Defined by the OpenID provider (in this case Google). It is the endpoint OrderCloud will redirect user to in order to validate credentials.                                                                                |
+    | `TokenEndpoint`         | Defined by the OpenID provider (in this case Google). It is the endpoint OrderCloud will call out to get a token from the provider.                                                                                        |
+    | `UrlEncoded`            | How to post information to the OpenID provider (in this case Google). It is sent with either Basic Auth if UrlEncoded is `false`, otherwise it posts a Url encoded body. Most providers (such as Google) will accept both. If you encounter an error with your provider a good first step for troubleshooting is changing this value to the opposite of what its set. |
+    | `CallSyncUserIntegrationEvent`            | Whether or not the /syncuser endpoint will be called which is used to update user details that may have changed after their initial login |
+    | `IntegrationEventID`    | The ID to the Integration Event created in <Link to="/knowledge-base/implementing-single-sign-on-with-google-idp#2-create-the-openid-connect-integration-event">step 2.2</Link>. This has information about which endpoint OrderCloud should call out to in order to create the user after the user has successfully logged in.                                                                                                                                   |
+    | `AdditionalIdpScopes`   | As defined by the OIDC specification we will request profile, email, and oidc scope but you may request any additional scopes you'd like to request from the IDP at the time of authentication. As an example, you could request permissions from google to access user's google drive files, then the access token you get back from the IDP would have permission to do that. Please note that these roles will show up in the user consent screen and best practices dictate to only request those that you absolutely need for your application       
```

Comment:
profile, email, and OIDC* scope, Google* in two places

As an example, you could request permissions from Google* to access the* user's Google* Drive* files

Reply:
Keeping oidc lowercase because the scope is case sensitive but made the rest of changes